changelogs/unreleased/changelogs/unreleased/8304-update-installation-instructions-native-docker-container.yml

```diff
@@ -0,0 +1,9 @@
+description: Update installation instructions for new native docker container
+issue-nr: 8304
+change-type: patch
+destination-branches: [master]
+sections:
+  - upgrade-note: >
+      The iso 8 orchestrator image introduces a number of changes, including these breaking changes:
```

Comment:
This is the changelog entry for OSS. So we cannot talk about iso8 here. I would use the actual name of the image here.